clients/auth/auth.go

```diff
@@ -152,10 +152,21 @@ func (c Client) Create(ctx context.Context, params *CreateParams) error {
 			actions = append(actions, WriteAction)
 		}
 
+		permOrgID := orgID
+		// Orgs and users are both "top-level" resources, not nested under any other org.
+		// Read/write permissions on those resource types must also be top-level to have an effect.
+		switch resourcePermission.Name {
+		case string(extras.RESOURCEENUMCLOUD_ORGS):
+			permOrgID = ""
+		case string(extras.RESOURCEENUMCLOUD_USERS):
```

Comment:
If we're dealing with the users permission properly, should we also deal with non-operator users that want permission to edit themselves but not other users?

Reply:
(see special handling for the all-access permissions)

Reply:
I think I'd prefer to punt those situations to a follow-up PR and generalize the current special case of `--<action>-bucket <id1>,<id2>,...` to a set of `--<action>-<type> <id1>,<id2>,...` flags